### ISSUE TITLE
[Android] An app with POSIX wrapper crashes if launched in debug mode

### ISSUE THREAD

**Issue by proninyaroslav** [aldenml/libtorrent4j#49]
Hello. This happens in Android Studio (current version 3.4), Android 8.1 and below. Tested on Android `8.1` (`arm64`, `x86`), `7.1.1` (`x86`) and `4.4` (`x86`). Android 9 hasn't this problem, tested on `x86` and `x86_64` architecture.

Steps to reproduce:
1) Launch an app in debug mode
2) Do some file I/O in `libtorrent` with default POSIX wrapper (a child class, that calls only `super` methods, for example `super.stat()`)
3) Crash occurs:
```
A/libc: Fatal signal 11 (SIGSEGV), code 1, fault addr 0x0 in tid 10500 (Thread-15), pid 10377 (retorrent.debug)

A: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
A: LineageOS Version: '15.1-20190224-NIGHTLY-angler'
A: Build fingerprint: 'google/angler/angler:8.1.0/OPM7.181205.001/5080180:user/release-keys'
A: Revision: '0'
A: ABI: 'arm64'
A: pid: 10039, tid: 10230, name: Thread-17  >>> org.proninyaroslav.libretorrent.debug <<<
A: signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0
A: Cause: null pointer dereference
A:     x0   0000000000000000  x1   0000000000040000  x2   00000077d9d26e30  x3   0000000000000010
A:     x4   00000077d9d26d90  x5   0000000000000000  x6   0000000000000000  x7   00000077dccadfc0
A:     x8   00000077dcbf14a0  x9   0000000000000002  x10  0000000000000002  x11  0000000000000000
A:     x12  00000077dccae140  x13  00000077dccae170  x14  00000000ffffffff  x15  000000000000002b
A:     x16  000000787aa94240  x17  000000787a9e54d0  x18  000000000000002c  x19  00000077da4ea600
A:     x20  0000000000000010  x21  00000077d9d26e30  x22  0000000000000002  x23  0000000000040000
A:     x24  00000077ebe95600  x25  00000077d9d270b0  x26  0000000000000002  x27  0000000000000010
A:     x28  0000000000040000  x29  00000077d9d26d80  x30  00000077dc79342c
A:     sp   00000077d9d26d20  pc   00000077dc724278  pstate 0000000040000000
A: backtrace:
A:     #00 pc 000000000019c278  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #01 pc 000000000020b428  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #02 pc 000000000020dacc  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #03 pc 000000000020bbc8  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #04 pc 000000000020add8  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #05 pc 0000000000186cd8  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #06 pc 0000000000183a1c  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #07 pc 00000000001883c4  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #08 pc 00000000001886dc  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #09 pc 0000000000190f58  /data/app/org.proninyaroslav.libretorrent.debug-_cbSvDDA0hNLvZJd6s96sQ==/lib/arm64/libtorrent4j-1.2.1.0.so
A:     #10 pc 0000000000068e84  /system/lib64/libc.so (__pthread_start(void*)+36)
A:     #11 pc 000000000001ec30  /system/lib64/libc.so (__start_thread+68)
E: Tombstone written to: /data/tombstones/tombstone_39
```

**Comment by proninyaroslav** [comment]
1.2.2.0 still has this problem.

**Comment by proninyaroslav** [comment]
It seems to be fixed in 1.3.0-alpha-5.
